packages/app-project/stores/helpers/cookie.js

```diff
@@ -1,10 +1,15 @@
+import { compact } from 'lodash'
+
 // Borrowed from https://stackoverflow.com/a/25490531/5372931
 export function getCookie (name) {
   // process.browser doesn't exist in the jsdom test environment
   if (process.browser || process.env.BABEL_ENV === 'test') {
-    const value = document.cookie.match('(^|[^;]+)\\s*' + name + '\\s*=\\s*([^;]+)')
+    const match = document.cookie.match('(^|[^;]+)\\s*' + name + '\\s*=\\s*([^;]+)')
+
+    const compactMatchArray = compact(match)
```

Comment:
```suggestion
    const compactMatchArray = match.filter(Boolean)
```
does the same thing (removes all falsy values from `match`.)

`compactMatchArray` could be better named. `matchingCookieValues` maybe? I prefer array names that describe what the arrays are storing.